### PR TITLE
Add iCalendar 'sequence' handling

### PR DESCRIPTION
**Pull request by kanunnikau**
https://datatracker.ietf.org/doc/html/rfc5545#section-3.8.7.4

   Description:  When a calendar component is created, its sequence
      number is 0.  It is monotonically incremented by the "Organizer's"
      CUA each time the "Organizer" makes a significant revision to the
      calendar component.

Some calendar clients do not see updates (completion) submitted by SuperProductivity as the 'sequence' number is unchanged.

(I have not tested this, just wrote code blindly. Please review)

# Description

Makes SuperProductivity increment VTASK SEQUENCE property when marking as complete.

## Issues Resolved

~I didn't look for any reported issues :)~
Quick search did not yield any similar issues.

## Check List

- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by johannesjo** [comment]
Thank you very much for this fix! :)

File: src/app/features/issue/providers/caldav/caldav-client.service.ts [hunk 1]
```diff
@@ -402,6 +402,14 @@ export class CaldavClientService {
     }
     todo.updatePropertyWithValue('last-modified', now);
     todo.updatePropertyWithValue('dtstamp', now);
+
```
Comment (johannesjo) [inline src/app/features/issue/providers/caldav/caldav-client.service.ts:405]: Could you maybe also add a comment on why we need this?

File: src/app/features/issue/providers/caldav/caldav-client.service.ts [hunk 2]
```diff
@@ -402,6 +402,14 @@ export class CaldavClientService {
     }
     todo.updatePropertyWithValue('last-modified', now);
     todo.updatePropertyWithValue('dtstamp', now);
+    
+    var sequenceInt = 0;
+    const sequence = todo.getFirstPropertyValue('sequence');
+    if (sequence) {
+      sequenceInt = parseInt(sequence);
+    }
+    sequenceInt++;
```
Comment (johannesjo) [inline src/app/features/issue/providers/caldav/caldav-client.service.ts:411]:
```suggestion
    const sequence = todo.getFirstPropertyValue('sequence');
    const sequenceInt = sequence
      ? parseInt(sequence) + 1
      : 1;
```

File: src/app/features/issue/providers/caldav/caldav-client.service.ts (hunk 1 above)
Comment (kanunnikau) [reply src/app/features/issue/providers/caldav/caldav-client.service.ts:405]: Done

**Comment by johannesjo** [comment]
Thanks for the update! Could you please check again? There are still some linting errors.

**Comment by kanunnikau** [comment]
> There are still some linting errors

Done

**Pull request merged**